web/src/app/components/job-list-item/job-list-item.component.spec.ts

```diff
@@ -238,4 +251,32 @@ describe('JobListItemComponent', () => {
     // Expect 6 nodes since one is for the loi and 5 for submissions
     expect((await job.getNodes()).length).toBe(6);
   });
+
+  it('should highlight LOI when it is selected', () => {
+    const lois = createLois(1);
+    lois$.next(lois);
+
+    locationOfInterestId$.next(lois.first()!.id);
+    fixture.detectChanges();
+
+    const loiNodeElement = fixture.debugElement.query(By.css('mat-tree-node'));
+    expect(loiNodeElement.classes).toEqual(
+      jasmine.objectContaining({'tree-node-selected': true})
+    );
+  });
+
+  it('should select LOI when LOI is clicked', () => {
+    const lois = createLois(1);
+    lois$.next(lois);
+    const loiId = lois.first()!.id;
+
+    const selectLoiButton = fixture.debugElement.query(
+      By.css('mat-tree-node #select-loi')
+    ).nativeElement as HTMLElement;
+    selectLoiButton.click();
```

Comment:
this would become `await selectLoiButton.click();` once the button is a test harness

Reply:
Done.